src/pages/Home.jsx

```diff
@@ -1,6 +1,136 @@
+                <div className="relative z-10 flex flex-col items-center justify-center h-full text-black">
+                    <h2 className="text-4xl font-bold">JOIN FUSION</h2>
+                    <p className="text-2xl mt-4">WE ARE ALWAYS MORE OPEN TO PEOPLE JOINING US.</p>
+                    <button className="fusion-button mt-6">BUTTON</button>
+                </div>
+            </div>
+            {/* Footer */}
+            <footer className="footer bg-yellow-500 py-8">
+                <div className="container mx-auto flex flex-col items-center text-center">
+                    <div className="flex justify-between w-full mb-4">
+                        <div className="flex items-center space-x-4">
+                            <a href="#contact" className="text-black">CONTACT US</a>
+                            <a href="mailto:fusion@uci.edu" className="text-black">FUSION@UCI.EDU</a>
+                        </div>
+                        <div className="flex items-center space-x-4">
+                            <a href="#home" className="text-black">HOME</a>
+                            <a href="#about" className="text-black">ABOUT</a>
+                            <a href="#meetings" className="text-black">MEETINGS</a>
+                            <a href="#board" className="text-black">BOARD</a>
+                            <a href="#programs" className="text-black">PROGRAMS</a>
+                            <a href="#fusioncon" className="text-black">FUSIONCON</a>
+                        </div>
+                    </div>
+                    <div className="flex justify-center space-x-4 mb-4">
+                        <a href="https://www.instagram.com/fusionatuci/" className="text-black " target="_blank" rel="noopener noreferrer">
+                            <img src={InstagramLogo} alt="Instagram" className="h-6" />
+                        </a>
+                        <a href="https://discord.gg/9wnrCWEfxK" className="text-black" target="_blank" rel="noopener noreferrer">
+                            <img src={DiscordLogo} alt="Discord" className="h-6" />
+                        </a>
+                        <a href="https://www.facebook.com/fusionuci/" className="text-black" target="_blank" rel="noopener noreferrer">
+                            <img src={FacebookLogo} alt="Facebook" className="h-6" />
+                        </a>
+                        <a href="https://www.linkedin.com/in/fusionatuci/" className="text-black" target="_blank" rel="noopener noreferrer">
+                            <img src={LinkedInLogo} alt="LinkedIn" className="h-6" />
+                        </a>
+                    </div>
+                    <div className="text-center mt-4 text-black">
+                        &copy; Copyright FUSION 2024
+                    </div>
+                </div>
+            </footer>
```

Comment:
I appreciate your work on this but you can delete this since Ethan is making the footer